onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -1721,6 +1720,71 @@ TEST_F(GraphTransformationTests, BiasGeluTest) {
   ASSERT_TRUE(op_to_count["BiasGelu"] == 1);
 }
 
+TEST_F(GraphTransformationTests, BiasGeluSwitchedInputOrder) {
+  auto model_uri = MODEL_FOLDER "fusion/bias_gelu_fusion_format_2.onnx";
```

Comment:
A comment for this test with some of the info from the PR description would be helpful for people to understand what exactly is being validated.